### PR TITLE
Bug 1200539 - Sync app integration tests, r=ferjm

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e5aec490ae7b8056c9b98aae1d14e3533344d491

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=430e2a6e15dda6e1f2a153587c6f664f2341f63b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a1c5d69a903bdf89a2a38343c758c18510a1c97e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ec76c3a112273dbe265f26c3b234cae9bf73e4f0

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c4f8234af48d72e3d72edb880e892a039d8eadfb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f5a10e73f6f4b02b292a05c85311fa693075671a

File: apps/sync/test/unit/integration_test.js [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+  suite,
+  SynctoServerFixture,
+  teardown,
+  test
+*/
+
+
+requireApp('system/test/unit/mock_iac_handler.js');
+require('/shared/test/unit/mocks/mock_navigator_datastore.js');
+// require('/apps/homescreen/test/unit/mocks/mock_async_storage.js');
```
Comment (ferjm) [inline apps/sync/test/unit/integration_test.js]: Remove this line, please.

File: apps/sync/test/unit/integration_test.js [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+      }));
+
+      windowClosed('history').then(() => {
+        expect(postMessageSpy.called).to.equal(true);
+        expect(postMessageSpy.args[0][0].id).to.equal(id);
+        expect(postMessageSpy.args[0][0].error).to.equal(undefined);
+        var fetchArgsExpected = SynctoServerFixture.fetchArgsExpected(
+            ['meta', 'crypto', 'history']);
+        // FIXME: We are not mocking IndexedDB, so it's possible the crypto
+        // crypto collection was cached in the test environment and does not
```
Comment (ferjm) [inline apps/sync/test/unit/integration_test.js]: "crypto crypto" :)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=37b8108a509929231408d39bcd9d82b8f7501b44

**Pull request merged**